stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -638,6 +573,9 @@ public class MessageListView : ConstraintLayout {
 
     public fun setNewMessagesBehaviour(newMessagesBehaviour: NewMessagesBehaviour) {
         this.newMessagesBehaviour = newMessagesBehaviour
+        if (::scrollHelper.isInitialized) {
```

Comment:
Is there a chance that it's not? As far I see, we'll always initialize it in the constructor.